code/src/Core/Locations/RemoteTemplatesSource.cs

```diff
@@ -63,11 +72,9 @@ private static void DownloadContent(string sourceUrl, string file)
             }
             catch (Exception ex)
             {
-                string msg = "The templates content can't be downloaded.";
-
-                AppHealth.Current.Exception.TrackAsync(ex, msg).FireAndForget();
-
-                throw;
+                string msg = $"Templates content can't be downloaded right now, we will try it later.";
```

Comment:
yes, it is the case, it will try to download it again next time WTS is used (if have passed at least 120 mins from last download)